plugins/inputs/pgbouncer/pgbouncer.go

```diff
@@ -0,0 +1,196 @@
+package pgbouncer
+
+import (
+	"bytes"
+	"database/sql"
+	"regexp"
+	"strings"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+
+	"github.com/lib/pq"
```

Comment:
@james-lawrence I have tried your suggestion of not using "version", but the error happens either way:

`$ telegraf --config /etc/telegraf/telegraf.d/99-pgbouncer.pgbouncer.conf --test
* Plugin: inputs.postgresql_extensible, Collection 1
* Internal: 1m0s
2017-08-21T16:25:19Z E! ERROR: unsupported pkt type: 80 (SQLSTATE 08P01)`